lib/open_food_network/variant_stock.rb

```diff
@@ -125,5 +134,12 @@ def overwrite_stock_levels(new_level)
       # have only one stock location.
       stock_items.first.__send__(:count_on_hand=, new_level)
     end
+
+    def warn_deprecation(method_name, new_method_name)
```

Comment:
I'm feeling a bit uneasy to introduce deprecation warnings without having a clear plan what else to use. We can refer to the stock items, but that may change again when we detailed the network feature. I would only deprecate the methods we are sure how to replace them.